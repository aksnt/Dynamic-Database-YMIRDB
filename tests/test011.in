SET a 1
SET b a
SET c b
FORWARD a
FORWARD c
SET d a b c
BACKWARD c
BACKWARD d
BYE
