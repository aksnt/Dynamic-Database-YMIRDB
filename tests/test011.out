> ok

> ok

> ok

> nil

> a, b

> ok

> d

> nil

> bye
